project_event/models/mail_thread.py

```diff
@@ -35,7 +35,8 @@ def _message_track(self, tracked_fields, initial):
                 col_info['string'] = _('Created by')
 
             if col_name == 'notes':
-                col_info['string'] = _('Note was created/modified')
+                col_info['string'] = _('Note')
+                new_value = 'created/modified'
```

Comment:
@eilst  : What is the purpose? In there any anomaly or change request for this task.